theme_kea/views/snippets/s_color_blocks_2.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="s_color_blocks_2" inherit_id="website.s_color_blocks_2">
```

Comment:
For all new files, try to follow our correct convention: no indentation and a blank line after odoo

```xml
<odoo>

<template>
[...]
</template>

</odoo>
```